posthog/special_migrations/migrations/example.py

```diff
@@ -1 +1,112 @@
+from ee.clickhouse.client import sync_execute
```

Comment:
Maybe put this file in `special_migrations/migrations_examples/` directory and use that dir for tests rather than removing a single example file in prod

Reply:
I have this here because on debug you will have this migration to use and test - it's not just a visual example

Reply:
I could add it though I guess

Reply:
I recalled you had some logic in the original large pr that explicitly removed this example from the list of migrations to look at.

Reply:
Yup, exactly, hence it can sit here.

My point is I guess I could change to your approach and import the submodule from another path if DEBUG is on, but that'd be some more work.